src/compiler/diagnosticMessages.json

```diff
@@ -751,8 +751,22 @@
         "category": "Error",
         "code": 1235
     },
-
-
+    "'{0}' modifier can only appear on a class or member function declaration.": {

```

Comment:
no need for the argument. this is only used with "abstract" if we needed it for some other modifier we can parameterize it then.